product_docs/docs/pgd/5/durability/group-commit.mdx

```diff
@@ -124,46 +124,44 @@ does eventually COMMIT even though the client might receive an abort message.
 
 See also [Limitations](limitations). 
 
-### Transaction Reconciliation
+### Transaction reconciliation
 
-A group commit transaction’s commit on the origin node is implicitly converted 
-into a two phase commit. 
+A Group Commit transaction's commit on the origin node is implicitly converted 
+into a two-phase commit. 
 
-In the first phase (prepare) the transaction is prepared locally and made ready to commit.
-The data is made durable but uncomitted at this stage so other transactions
-cannot see the changes made by this transaction. This prepared transaction gets
+In the first phase (prepare), the transaction is prepared locally and made ready to commit.
+The data is made durable but is uncomitted at this stage, so other transactions
+can't see the changes made by this transaction. This prepared transaction gets
 copied to all remaining nodes through normal logical replication. 
 
-The origin node seeks confirmations from other nodes, as per rules in the group
-commit grammar. If it gets confirmations from minimum required nodes in the
-cluster, it decides to commit this transaction moving onto the second phase (commit) 
-where it also sends this decision via replication to other nodes which will also eventually commit on getting this message.
+The origin node seeks confirmations from other nodes, as per rules in the Group
+Commit grammar. If it gets confirmations from the minimum required nodes in the
+cluster, it decides to commit this transaction moving onto the second phase (commit). 
+In the commit phase, it also sends this decision by way of replication to other nodes. Those nodes will also eventually commit on getting this message.
 
-There is a possibility of failure at various stages. For example, the origin
+There's a possibility of failure at various stages. For example, the origin
 node may crash after preparing the transaction. Or the origin and one or more
-replicas may crash. 
+replicas may crash.
 
 This leaves the prepared transactions in the system. The `pg_prepared_xacts`
 view in Postgres can show prepared transactions on a system. The prepared
-transactions could be holding locks and other resources and they therefore need
-to be either aborted or committed. That decision has to be made with a consensus
+transactions might be holding locks and other resources and they therefore need
```

Comment:
Wasn't sure if it should be "can," "might," or maybe it should be recast as: It's possible for the prepared transactions to be holding locks and other resources. They therefore...

Now that I'm looking at this again, I think that's a better edit and that a separate sentence isn't a bad idea. What think?

Reply:
I think that's a bit clearer in the active voice.